packages/client/src/features/tools/edge-creation/edge-creation-tool.ts

```diff
@@ -147,6 +148,31 @@ export class EdgeCreationToolMouseListener extends DragAwareMouseListener {
         return result;
     }
 
+    /**
+     * Determines wether the tool should run in continuous mode (also called stamp mode) or not.
+     * If continuous mode is enabled, the tool will stay after a successful creation.
+     * The user can then create more elements of the same type without having to re-trigger the tool.
+     * By default, continuous mode is enabled if the user holds the CTRL key.
+     * @param element the current model element
+     * @param event
+     */
+    protected isContinuousMode(element: GModelElement, event: MouseEvent): boolean {
+        return isCtrlOrCmd(event);
+    }
+
+    protected getCreateOperation(element: GModelElement, event: MouseEvent, sourceElementId: string, targetElementId: string): Action {
```

Comment:
I think we are missing the call of this method in our `nonDraggingMouseUp` handling.